diffkemp/simpll/DebugInfo.cpp

```diff
@@ -479,3 +479,40 @@ void DebugInfo::removeFunctionsDebugInfo(Module &Mod) {
     for (auto &F : Mod)
         fpm.run(F, fam);
 }
+
+DICompositeType *getVariableTypeInfo(Value *Val) {
+    if (!Val->getType()->isIntegerTy())
+        return nullptr;
+
+    if (isa<Argument>(Val) || isa<AllocaInst>(Val)) {
```

Comment:
Add comment: "Val must be a parameter or a local variable so that we are able to extract its debug info"